examples/rest_api/rest_api.py

```diff
@@ -0,0 +1,134 @@
+import dbinfo
+import requests
+import json
+
+URL = "http://" + dbinfo.DB_HOST + '/api'
+
+
+def dict_query_to_str(dict_query):
+
+    return str(json.dumps(dict_query)).replace("'", '\"')
```

Comment:
the replace is not required, json.dumps converts to JSON with double quotes. Is it for something specefic?

Reply:
we need the double quotes to be scaped, as que query is passed as a string.